qa-tester/patches/@reach+utils+0.14.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@reach/utils/dist/reach-utils.esm.js b/node_modules/@reach/utils/dist/reach-utils.esm.js
```

Comment:
This is needed to fix an issue with the build selector where options were not able to be selected due to the component being rendered in a shadow DOM.